### DISCUSSION TITLE
The composite image color changes

### DISCUSSION THREAD

**Discussion by jueqing1015** [dlemstra/Magick.NET#709] [Help]
I created a new MAGICKIMAGE and composited it on another picture, but its color changed and the color area was yellow
MagickColor Color=MagickColors.White
   MagickImage image = new MagickImage(Color, rect.Width, rect.Height);     
 image.Density = MagickPic.Density;                
   MagickPic.Composite(image, rect.Left, rect.Top);
     MagickPic.Write("c:\\jpg.jpg",MagickFormat.Jpeg);    ////The image color is normal after saving          
                PictureBox.Image = MagickPic.ToBitmap();//Converted to BITMAP and displayed in PictureBox The image color is abnormal
NO。1  Original image
NO.2  Images saved locally
NO. 3 Image displayed on PICTUREBOX
![0035-1](https://user-images.githubusercontent.com/65677359/89196319-a2425900-d5dc-11ea-8b71-56729b206f59.jpg)
![jpg](https://user-images.githubusercontent.com/65677359/89196509-e46b9a80-d5dc-11ea-981d-dcabe3f2a755.jpg)
![j1](https://user-images.githubusercontent.com/65677359/89196499-e2094080-d5dc-11ea-83d3-d8869ce79423.jpg)

@dlemstra

**Comment by dlemstra** [comment]
I am unable te reproduce your issue. Is it possible to create a super small program that demonstrates the issue and publish it on GitHub?

**Reply by jueqing1015** [reply]
[WindowsFormsApp1.zip](https://github.com/dlemstra/Magick.NET/files/5035542/WindowsFormsApp1.zip)
.net 4.7.2
@dlemstra

**Reply by dlemstra** [reply]
Thanks for reporting this and providing me with an example. Your code made it clear that I created an incorrect optimization for Grayscale images. I have fixed this and this will be resolved in the next release of Magick.NET. For now you could do `image.ColorSpace = ColorSpace.sRGB` to resolve your issue.

**Reply by dlemstra** [reply]
A new release has just been published that resolves this issue.
